### PR TITLE
Fix detecting libraries included inside mods; Add texture pack button to pause screen

### PR DESCRIPTION
**Pull request by telvarost**
Also fixed a bug with mod description not appearing, since text renderer was null

### CODE REVIEW

File: src/main/java/io/github/prospector/modmenu/mixin/MixinGameMenuScreen.java [hunk 1]
```diff
@@ -17,12 +20,29 @@ public class MixinGameMenuScreen extends Screen {
 	@SuppressWarnings("unchecked")
 	@Inject(at = @At("RETURN"), method = "init")
 	public void drawMenuButton(CallbackInfo info) {
-		this.buttons.add(new ModMenuButtonWidget(100, this.width / 2 - 100, this.height / 4 + 72 - 16, 200, 20,  "Mods (" + ModMenu.getFormattedModCount() + " loaded)"));
+		if (FabricLoader.getInstance().isModLoaded("legacytranslations")) {
+			TranslationStorage var2 = TranslationStorage.getInstance();
+			this.buttons.add(new ButtonWidget(100, this.width / 2 - 100, this.height / 4 + 72 - 16, 98, 20, var2.get("menu.texturepacks")));
+			this.buttons.add(new ModMenuButtonWidget(101, this.width / 2 + 2, this.height / 4 + 72 - 16, 98, 20, var2.get("menu.mods") + " (" + ModMenu.getFormattedModCount() + ")"));
```
Comment (thecatcore) [inline src/main/java/io/github/prospector/modmenu/mixin/MixinGameMenuScreen.java]: Where did the "loaded" part go?

File: src/main/java/io/github/prospector/modmenu/mixin/MixinTitleScreen.java [hunk 2]
```diff
@@ -18,14 +20,26 @@ public class MixinTitleScreen extends Screen {
...
-		int newWidth = ((MixinGuiButton) texturePackButton).getWidth() / 2 - 1;
-		((MixinGuiButton) texturePackButton).setWidth(newWidth);
-		this.buttons.add(new ModMenuButtonWidget(100, this.width / 2 + 2, texturePackButton.y, newWidth, 20,  "Mods (" + ModMenu.getFormattedModCount() + " loaded)"));
+
+		if (FabricLoader.getInstance().isModLoaded("legacytranslations")) {
+			TranslationStorage var2 = TranslationStorage.getInstance();
+			texturePackButton.text = var2.get("menu.texturepacks");
+			int newWidth = ((MixinGuiButton) texturePackButton).getWidth() / 2 - 1;
+			((MixinGuiButton) texturePackButton).setWidth(newWidth);
+			this.buttons.add(new ModMenuButtonWidget(100, this.width / 2 + 2, texturePackButton.y, newWidth, 20,  var2.get("menu.mods") + " (" + ModMenu.getFormattedModCount() + ")"));
```
Comment (thecatcore) [inline src/main/java/io/github/prospector/modmenu/mixin/MixinTitleScreen.java]: Same here

**Review by thecatcore — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/prospector/modmenu/mixin/MixinGameMenuScreen.java (hunk 1 above)
Comment (telvarost) [reply src/main/java/io/github/prospector/modmenu/mixin/MixinGameMenuScreen.java]: Added back in, and upgraded LegacyTranslations to be able to handle button text overflow

File: src/main/java/io/github/prospector/modmenu/mixin/MixinTitleScreen.java (hunk 2 above)
Comment (telvarost) [reply src/main/java/io/github/prospector/modmenu/mixin/MixinTitleScreen.java]: Added back in, and upgraded LegacyTranslations to be able to handle button text overflow

**Comment by calmilamsy** [comment]
Looks good.

**Pull request merged**